lib/project/ext/ui_table_view_controller.rb

```diff
@@ -0,0 +1,13 @@
+class UITableViewController
+  # initWithStyle is called before viewDidLoad, if we use the stylesheet in
+  # the table_data, we need to set the stylesheet earlier.
+  alias :original_initWithStyle initWithStyle
```

Comment:
I had a test failing on master and this fixed it. But yeah they should probably be consistent. Didn't notice I added that first ':'